src/harness/sas_interface.py

```diff
@@ -339,4 +354,21 @@ def TriggerMeasurementReportHeartbeat(self, request):
     Note: The SAS should request a measurement report in the HeartbeatResponse 
     (if status == 0)
     """
+    pass
+
+class SasTestcaseInterface(object):
+  """Includes Helper Function interface for SAS-CBSD and SAS-SAS Testcases"""
+
+  __metaclass__ = abc.ABCMeta
+
+  @abc.abstractmethod
+  def AssertContainsRequiredFields(self, schema_path=None, response=None):
+    """Assertion of Required Fields in Response validating it with Schema
+
+    Args:
+      schema_path: A string containing the filename of the schema to be used
```

Comment:
Maybe it's better to call it "schema_filename"? Otherwise they might think they need to provide the full path.

Do we need to tell the user where this function will look for the schema files?